src/transformers/configuration_blenderbot.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# coding=utf-8
+# Copyright (c) Facebook, Inc. and Huggingface, 2020
+#
+# This source code is licensed under the MIT license found in the;
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# LICENSE file in the root directory of this source tree.
+from .configuration_bart import BartConfig
+
+
+BLENDERBOT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "facebook/blenderbot-3B": "https://cdn.huggingface.co/facebook/blenderbot-3B/config.json",
+    "facebook/blenderbot-90M": "https://cdn.huggingface.co/facebook/blenderbot-90M/config.json",
+}
+
+
+class BlenderbotConfig(BartConfig):
+    """
+    This class overrides :class:`~transformers.BartConfig`. Please check the superclass for the appropriate documentation alongside usage examples.
```

Comment:
No this should have the documentation with the proper defaults specific to `BlenderbotConfig`, like all other model configs.

Reply:
There are two sets of defaults, one for blenderbot-90 and one for blenderbot-3B. If you call `__init__` directly (not encouraged anywhere) the defaults are the same as bart.

Reply:
Every model has various set of defaults for the different versions, which is why the other config docstrings have a sentence like: "Instantiating a configuration with the defaults will yield a similar configuration to that of the BERT `bert-base-uncased <https://huggingface.co/bert-base-uncased>`__ architecture." This should have the same thing here, after selecting the most simple of the architectures.

The `__init__` is another issue which I have already pointed out before, but if you want to go down that road again, we certainly cam. Instantiating each of these (=BART, mBART, Pegasus, Marian and now BlenderBot) config without the `from_pretrained` method should yield a proper config object that match the model, as is the case for all other models of the library in case a user wants to retrain one of these architectures on its own dataset.